firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/FilterDialogFragment.kt

```diff
@@ -7,27 +7,21 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.DialogFragment
 import com.google.firebase.example.fireeats.R
+import com.google.firebase.example.fireeats.databinding.DialogFiltersBinding
 import com.google.firebase.example.fireeats.kotlin.model.Restaurant
 import com.google.firebase.firestore.Query
-import kotlinx.android.synthetic.main.dialog_filters.spinnerCategory
-import kotlinx.android.synthetic.main.dialog_filters.spinnerCity
-import kotlinx.android.synthetic.main.dialog_filters.spinnerPrice
-import kotlinx.android.synthetic.main.dialog_filters.spinnerSort
-import kotlinx.android.synthetic.main.dialog_filters.view.buttonCancel
-import kotlinx.android.synthetic.main.dialog_filters.view.buttonSearch
 
 /**
  * Dialog Fragment containing filter form.
  */
 class FilterDialogFragment : DialogFragment() {
 
-    private lateinit var rootView: View
-
+    private lateinit var binding: DialogFiltersBinding
```

Comment:
For Fragments, the [Android Docs](https://developer.android.com/topic/libraries/view-binding#fragments) recommend holding a nullable reference and a non-null one to make sure it is tied to the Fragment's lifecycle. So this would become:

```kotlin
private var _binding: DialogFiltersBinding? = null
private val binding get() = _binding!!
```
and then we should override `onDestroyView()` to set  `_binding` back to `null`:
override fun onDestroyView() {
    super.onDestroyView()
    _binding = null
}